src/authn/ccloudStateHandling.test.ts

```diff
@@ -179,59 +177,38 @@ describe("authn/ccloudPolling.ts watchCCloudConnectionStatus()", () => {
   beforeEach(() => {
     sandbox = sinon.createSandbox();
 
-    getCCloudConnectionStub = sandbox.stub(connections, "getCCloudConnection");
     nonInvalidTokenStatusFireStub = sandbox.stub(nonInvalidTokenStatus, "fire");
 
     stubResourceManager = sinon.createStubInstance(ResourceManager);
     sandbox.stub(ResourceManager, "getInstance").returns(stubResourceManager);
-
-    pollFastStub = sandbox.stub(pollCCloudConnectionAuth, "useFastFrequency");
-    pollSlowStub = sandbox.stub(pollCCloudConnectionAuth, "useSlowFrequency");
   });
 
   afterEach(() => {
     sandbox.restore();
   });
 
-  it("should take no action when no connection is fetched from the sidecar", async () => {
```

Comment:
Drop test. The new equivalent to 'there is no ccloud connection at all' is for reactToCCloudAuthState() to never get called, period.